samza-sql/src/main/java/org/apache/samza/sql/SamzaSqlInputTransformer.java

```diff
@@ -40,7 +39,8 @@
   public Object apply(IncomingMessageEnvelope ime) {
     Assert.notNull(ime, "ime is null");
     KV<Object, Object> keyAndMessageKV = KV.of(ime.getKey(), ime.getMessage());
-    SamzaSqlRelMsgMetadata metadata = new SamzaSqlRelMsgMetadata(ime.getEventTime(), ime.getArrivalTime(),0L);
+    SamzaSqlRelMsgMetadata metadata =
+        new SamzaSqlRelMsgMetadata(ime.getEventTime(), ime.getArrivalSystemNanoTime(),0L);
```

Comment:
Minor: Please add whitespace before the final argument.

Reply:
done